schema/2.0.0/corpora_schema.md

```diff
@@ -0,0 +1,257 @@
+[gencode.vM23.primary_assembly.annotation.gtf]:http://ftp.ebi.ac.uk/pub/databases/gencode/Gencode_mouse/release_M23/gencode.vM23.primary_assembly.annotation.gtf.gz"
+
+[cellranger 2020-A (July 7, 2020) release]: https://support.10xgenomics.com/single-cell-gene-expression/software/release-notes/build
+
+## `obs` (Cell Metadata)
+
+`obs` is a [`pandas.DataFrame`](https://pandas.pydata.org/pandas-docs/stable/reference/api/pandas.DataFrame.html).
+
+Curators MUST annotate the following columns in the `obs` dataframe:
+
+| Key | Value | Annotator |
+|-|-|-|
+| assay_ontology_term_id | categorical with `str` categories. This MUST be a child of "experimental process". See [EFO:0002694](https://www.ebi.ac.uk/ols/ontologies/efo/terms?short_form=EFO_0002694) for details. If there is not an exact match for the assay, *clarifying text* MAY be appended in parentheses such as `" (sci-plex)"`. | Curator |
+| cell_type_ontology_term_id | categorical with `str` categories. This MUST be a CL term. | Curator |
+| development_stage_ontology_term_id | categorical with `str` categories. If unavailable, this MUST be `"unknown"`. <br><br> If `organism_ontolology_term_id` is `"NCBITaxon:9606"`, this MUST be a HsapDv term.<br><br> If `organism_ontolology_term_id` is `"NCBITaxon:10090"`, this MUST a MmusDv term with the following restrictions: <br><br> **Prenatal stages** MUST be in the range beginning with "MmusDv:0000003" and ending with "MmusDv:0000035". <br><br> **Postnatal stages (1-4 weeks)** MUST be in the range beginning with "MmusDv:0000045" and ending with "MmusDv:0000048". <br><br> **Postnatal stages (after week 4)** MUST be in the range beginning with "MmusDv:0000062" and ending with "MmusDv:0000091". | Curator |
+| disease_ontology_term_id | categorical with `str` categories. This MUST be a MONDO term or [PATO:0000461](http://purl.obolibrary.org/obo/PATO_0000461) for "normal". | Curator |
+| ethnicity_ontology_term_id | categorical with `str` categories. If `organism_ontolology_term_id` is `"NCBITaxon:9606"`, this MUST be either a HANCESTRO term or `"unknown"` if unavailable. <br><br> If `organism_ontolology_term_id` is `"NCBITaxon:10090"`, this MUST be `"na"`. | Curator |
+| is_primary_data | `bool`. This MUST be `True` if this is the canonical instance of this cellular observation and `False` if not. This is commonly `False` for meta-analyses reusing data or for secondary views of data. | Curator |
+| organism_ontology_term_id | categorical with `str` categories. This MUST be either `"NCBITaxon:9606"` or `"NCBITaxon:10090"`. | Curator |
+| sex_ontology_term_id | categorical with `str` categories. This MUST be a child of "phenotypic sex". See [PATO:0001894](http://purl.obolibrary.org/obo/PATO_0001894) for details. | Curator |
+| tissue_ontology_term_id | categorical with `str` categories. This MUST be an UBERON term. This SHOULD be appended with `" (cell culture)"` or `" (organoid)"` if appropriate. | Curator |
```

Comment:
@ambrosejcarr and @pablo-gar ?

Reply:
Thanks @jahilton; this is a carry over from when we were attempting to model "brain organoid" or "brain cell line". You correctly identify that the latter is certainly insufficiently specific. 

So, two questions: 
1. Would you suggest we model organoids and CL separately? 
2. How would you recommend modeling them? And in the case of cell lines, can we do better than a term from {CL, EFO, BTO, CLO}? Would it be possible to pick one of those ontologies, or in pratice have you seen a need to use them all? How much resolution would be sacrificed if we were to pick the ontology that's the best fit?

Reply:
organoids - I think organoids makes sense, as proposed. Those map to UBERON terms, and you have a nice/simple way of specifying "(organoid)"

cell culture/lines - I think you can keep the general concept here - fill in the `tissue_ontology_term_id` and append "(cell culture)", just enforce CL terms, as opposed to UBERON. Even if the culture is of a common cell line, I think the relevant information is the _type_ of cell it is, and the specific cell line can be noted in additional fields, dataset/collection description, attached paper, etc. (So if they used WTC-11, they would submit it as the `fibroblast` CL term). This also allows the researcher to differentiate a common cell line into a different cell type.

Reply:
Thanks @jahilton this guidance makes sense to me. How does this text sound? I think it incorporates your suggested changes, and also addresses cases where someone has, for example, sorted monocyte lineage and would prefer to use [myeloid leukocyte](https://www.ebi.ac.uk/ols/ontologies/cl/terms?iri=http%3A%2F%2Fpurl.obolibrary.org%2Fobo%2FCL_0000766&viewMode=PreferredRoots&siblings=false) rather than a less specific UBERON term.  

categorical with `str` categories. This MUST be the most specific UBERON or CL term. This SHOULD be appended with `" (cell culture)"` or `" (organoid)"` if appropriate.

Reply:
Sounds good

Reply:
@brianraymor suggested being more verbose about our guidance here. My recommendation: 

`tissue_ontology_term_id`: categorical with `str` categories. MUST be the term that best describes the tissue that this cell was derived from. Takes different values depending on the type of biological sample.

**Tissue:** MUST be an UBERON term (e.g. UBERON:0008930 for a sematosensory cortex tissue sample)
**Organoid:** MUST be an UBERON term appended with " (organoid)" (e.g. `UBERON:0000955 (organoid)` for a brain organoid)
**Sorted cells from a tissue:** MUST be a CL term (e.g. [CL:0000766](https://www.ebi.ac.uk/ols/ontologies/cl/terms?iri=http%3A%2F%2Fpurl.obolibrary.org%2Fobo%2FCL_0000766&viewMode=PreferredRoots&siblings=false) for sorted myeloid cells)
**Cell Culture:** MUST be a CL term appended with " (cell culture)" (e.g. `CL:0000057` for WTC-11)

Reply:
That @brianraymor is so demanding. Thanks Ambrose. 

> Cell Culture: MUST be a CL term appended with " (cell culture)" (e.g. CL:0000057 for WTC-11)

(e.g. **CL:0000057 (cell culture)** for the WTC-11 cell-line) ?

What are "sorted cells" ? Sounds like a _chocolate assortment_.

Reply:
Addressed in https://github.com/chanzuckerberg/single-cell-curation/pull/67/commits/b2d031d088a1300cc7056fb8503cba64bcf5dd04